content/docs/hooks-reference.md

```diff
@@ -321,18 +321,18 @@ function TextInputWithFocusButton() {
 
 Note that `useRef()` is useful for more than the `ref` attribute. It's [handy for keeping any mutable value around](/docs/hooks-faq.html#is-there-something-like-instance-variables) similar to how you'd use instance fields in classes.
 
-### `useImperativeMethods`
+### `useImperativeHandle`
 
 ```js
-useImperativeMethods(ref, createInstance, [inputs])
+useImperativeHandle(ref, createInstance, [inputs])
 ```
 
-`useImperativeMethods` customizes the instance value that is exposed to parent components when using `ref`. As always, imperative code using refs should be avoided in most cases. `useImperativeMethods` should be used with `forwardRef`:
+`useImperativeHandle` customizes the instance value that is exposed to parent components when using `ref`. As always, imperative code using refs should be avoided in most cases. `useImperativeHandle` should be used with `forwardRef`:
 
 ```js
 function FancyInput(props, ref) {
   const inputRef = useRef();
-  useImperativeMethods(ref, () => ({
+  useImperativeHandle(ref, () => ({
     focus: () => {
```

Comment:
Maybe we should change this to promote the static call style. You’ll need to discuss this with Seb.

Reply:
static call style? like, `React.` ? I'll reach out.

Reply:
No, I mean not using an object at all. Although I guess in that case `useImperativeHandle` would be unnecessary in the first place. Disregard.